src/storage/src/controller/hosts.rs

```diff
@@ -242,7 +265,7 @@ where
     }
 
     /// Starts a orchestrated storage host for the specified ID.
-    async fn start_storage_host(
+    async fn ensure_storage_host(
```

Comment:
Renamed to match the naming and semantics of the underlying Kube implementation a bit better. (ie. naming after the desired state and not the presumed action.)